themes/src/main/themes/VAADIN/themes/valo/components/_treegrid.scss

```diff
@@ -1,49 +1,43 @@
 @import "grid";
 
 /** Expander button visual - expanded */
-$tg-expander-char-expanded: '\f0d7' !default;
+$v-treegrid-expander-char-expanded: '\f0d7' !default;
 
 /** Expander button visual - collapsed */
-$tg-expander-char-collapsed: '\f0da' !default;
-
-/** Expander button width */
-$tg-expander-width: 10px !default;
-
-/** Expander button right side padding */
-$tg-expander-padding: 10px !default;
+$v-treegrid-expander-char-collapsed: '\f0da' !default;
 
 @mixin valo-treegrid ($primary-stylename: v-treegrid) {
 
   @include valo-grid($primary-stylename);
 
   // Expander with and item indentation constants
-  $indent: $tg-expander-width + $tg-expander-padding;
+  $indent:2ex !default;
 
   // Classname for depth styling
   $class-depth: depth !default;
```

Comment:
Same as for the `$indent`: this is a scoped variable, and I’m not sure you can override that from outside the mixin. So if this is meant to be as an API the developer can override, it should be moved to the top level scope and prefixed with `v-treegrid-`